Tests/Acceptance/ID3TagEditorTestAcceptanceNewFrames.swift

```diff
@@ -42,13 +45,16 @@ class ID3TagEditorTestAcceptanceNewFrames: XCTestCase {
         XCTAssertEqual(id3Tag?.frames[.Subtitle]?.id3Identifier, "TT3")
         XCTAssertEqual((id3Tag?.frames[.Subtitle] as? ID3FrameWithStringContent)?.content, "Subtitle V2")
         XCTAssertEqual(id3Tag?.frames[.UnsyncedLyrics]?.id3Identifier, "ULT")
+        #warning("The data in the “ULT” tag is “55 4C 54 00 00 05 00 00 10 00 00”. That appears to mean “ULT”, a 5‐byte size, a text encoding of Latin‐1, a language of “null, data escape, null”, and nothing more. But that is nonsense. The expected string is not present. What is actually in the file?")
```

Comment:
This is what is printed when analyzing the file with MediaInfo:

<img width="592" alt="Screen Shot 2020-03-19 at 1 06 07 AM" src="https://user-images.githubusercontent.com/56804260/77045240-080bac80-697e-11ea-87e8-7a7a642b4c96.png">

And here's the actual text:

> Lyrics                                   : 渀猀礀渀挀攀搀䰀礀爀椀挀猀 嘀㈀:  

(hopefully that won't cause this comment to get cut off.)

Reply:
All those Han characters have a byte pattern of `xx00`:

`6E00 7300 7900 6E00 6300 6500 6400 4C00 7900 7200 6900 6300 7300 2002 5600 3200`

Offset the sequence one byte to the right (or swap the endianness) and you get:

`??6E 0073 0079 006E 0063 0065 0064 004C 0079 0072 0069 0063 0073 0020 0256 0032 00??`

Which is this: “nsyncedLyrics ɖ2”. The tail end is likely some sort of data besides text.

Assuming it was supposed to start with “U” (or “u”), then it looks like an offset of three bytes (one and a half characters).

Reply:
To clarify, is this screenshot of a file generated with your earlier code, or of a file generated using my changes?

Reply:
That was generated with my code. I wasn't sure if yours was ready for a test run.